src/containers/auth/Form.tsx

```diff
@@ -4,41 +4,67 @@ import {
 	Keyboard,
 	KeyboardAvoidingView,
 	Platform,
+	StyleSheet,
 	TouchableWithoutFeedback,
 	View,
 } from 'react-native';
 import { Text } from 'react-native-elements';
 import { RootStackParamList } from '../../../App';
-import { styles } from './FormStyles';
 import { BackButton } from './FormUtils';
 
 type Props = {
 	backButton?: boolean;
-	title: string;
+	bigTitle?: boolean;
+	title?: string;
 	children: React.ReactNode;
 	navigation: StackNavigationProp<RootStackParamList, keyof RootStackParamList>;
 };
```

Comment:
Why are some props in a file like `PropTypes` in `types/mapTypes` and some are just declared locally?